src/test_host.cpp

```diff
@@ -185,8 +185,10 @@ void TestHost::SetVertexBufferAttributes(uint32_t enabled_fields) {
 
   // FIXME: Figure out what to do in cases where there are multiple stages with different swizzle flags.
   // Is this supported by hardware?
-  Vertex *vptr = texture_stage_[0].IsSwizzled() ? vertex_buffer_->normalized_vertex_buffer_
-                                                : vertex_buffer_->linear_vertex_buffer_;
+  Vertex *vptr = texture_stage_[0].IsSwizzled() ||
```

Comment:
If I am correct, the texture appears to be not swizzled but it still uses normalized texture coordinates.

Reply:
Interesting find! I would guess that the other `*L_DXT*` may follow the same pattern, in which case I think it might be better to just be explicit about the use of normalized/linear coords in the `TextureFormatInfo` struct.
 
If you're up for it, I think it'd be good to add a:
`bool xbox_linear{false};`

to that struct and populate it appropriately in `texture_format.cpp`. For all the entries other than the DXT ones it'd be == the `xbox_swizzle` setting.

Then we can add `IsLinear()` to `TextureStage` and use that here instead of `IsSwizzled`.

I can also do this in a followup if you don't have the time.

Reply:
You mean xbox_linear would be the opposite of the xbox_swizzled for all the other entries? Because only swizzled textures used normalized buffers before DXT1.

Reply:
Oops, yes, exactly, it'd be the inverse of `xbox_swizzled`, same as the existing behavior except for the DXT texture(s)